routes/api.php

```diff
@@ -17,4 +20,14 @@
     Route::apiResources([
         'profile' => UserProfileController::class,
     ]);
+
+    Route::apiResources([
+        // TODO: Event based routing?
+        'page' => PageController::class,
+    ]);
+
+    Route::apiResources([
+        // TODO: Event based routing?
+        'page_version' => PageVersionController::class,
+    ]);
```

Comment:
Does it make more sense to have the route as `/page/1/versions` ? 

```suggestion
    Route::apiResources([
        'profile' => UserProfileController::class,
        // TODO: Event based routing?
        'page' => PageController::class,
        'page.versions' => PageVersionController::class,
    ]);
```